### PR TITLE
refactor: Refine and fix

### PR DESCRIPTION
**Pull request by rajsam003**
### Description

This PR resolves the followings - 

- Move `CreateAlbum` folder from `sharedUI/screens` folder to `Albums` folder as its not a shared screen and belongs to only Albums tab. 
- Dont show Add To Album button in the artwork if the albums in empty. 
- Use `sharedUI/Header` in `HomeTabs.tsx`
- Remove `SuspenseWrapper` from `SelectPartner.tsx`
- Remove extra line in `src/app/sharedUI/items/index.ts`

### PR Checklist

- [x] I tested my changes on **iOS** and **Android**.
- [ ] I added screenshots or videos to illustrate my changes.
- [ ] I added Tests and Stories for my changes.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to run this PR on the simulator or device.

Need help with something? Have a look at our [docs], or get in touch with us.

[docs]: /docs/README.md

### CODE REVIEW

**Comment by rajsam003** [comment]
I moved the CreateAlbum folder from `sharedUI/screens` folder to `Albums` folder as its not a shared screen and belongs to only Album tab.

File: src/app/sharedUI/screens/Artwork/Artwork.tsx [hunk 1]
```diff
@@ -77,38 +77,40 @@ export const Artwork: React.FC<ArtworkProps> = () => {
...
-                  </Flex>
+                  <ArrowRightIcon ml="auto" fill="black100" />
                 )}
               </Flex>
-              {numberOfAlbumsTheArtworkAvailable === albums.length ? null : (
-                <ArrowRightIcon ml="auto" fill="black100" />
-              )}
-            </Flex>
-            <Spacer mt={3} />
-          </Touchable>
```
Comment (rajsam003) [inline src/app/sharedUI/screens/Artwork/Artwork.tsx:111]: This change is nothing but just this
image

VS code is smarter in showing the difference.

**Review by pvinis — APPROVED** [review]

File: src/app/sharedUI/screens/Artwork/Artwork.tsx (hunk 1 above)
Comment (pvinis) [reply src/app/sharedUI/screens/Artwork/Artwork.tsx:111]: using this little baby button fixes that ;)
Screenshot 2022-07-06 at 19 00 36

Comment (rajsam003) [reply src/app/sharedUI/screens/Artwork/Artwork.tsx:111]: I dont know why I dont have that?🤔

image

**Pull request merged**